.circleci/config.yml

```diff
@@ -83,14 +97,20 @@ jobs:
             helm repo add helm-gcs ${HELM_GCS_REPOSITORY}
             helm package --version ${CHART_VERSION} --app-version ${CHART_VERSION} ./helm/
             helm gcs push ${CHART_NAME}-${CHART_VERSION}.tgz helm-gcs --public --retry
+      - add_ssh_keys:
+          fingerprints:
+            - '18:7e:a0:7c:07:65:32:55:f5:35:a3:02:7f:88:25:21'
```

Comment:
I wonder why we need this and/or why this one arbitrarily.

Reply:
I see this is a copy paste so no need to b e addressed in this PR.

Reply:
The ssh key is to give write permission (default checks out with only read) to circleci before pushing a tag in the following step.

Reply:
Does this need to be public? Should it be a private variable in circle ci?

Reply:
The key itself is private, this is referencing it by fingerprint. This line, like any reference to a masked env var, would fail in build triggered by a user without write access.

Reply:
added the comment